### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by EmberCS15** [NJACKWinterOfCode/Wizzy-Downloader#5]
The project needs a proper demonstration of the installation procedure. Please update the readme.md file to incorporate all the required details of installation and what is required to get it running on a local system.

**Comment by Logan1x** [comment]
I Can update the readme but need your help because I am not Django friendly.

**Comment by EmberCS15** [comment]
Yeah Sure. Look up youtube to find tutorials on Django. It will just take about a day or two for you to get acquainted with Django. Or you can also add the .gitignore file for Django. Templates are available online.

**Comment by Logan1x** [comment]
ok, will try my best 😊.

**Comment by Mirnal1409** [comment]
Can I also work on this issue

**Comment by EmberCS15** [comment]
Yeah sure.

**Comment by the-codingninja** [comment]
Can this be more elaborated as I am not able to run the project locally.
And i am new to Django

**Comment by EmberCS15** [comment]
You can give it a try if you wish to work on this issue.

**Comment by the-codingninja** [comment]
Okay sure will give it a try but I need some information like how to install all pip requirements as none is present

**Comment by EmberCS15** [comment]
Install Python, then install pip and using pip install Django and pytube. That is all you will need.

**Comment by the-codingninja** [comment]
Yeah Finally got able to ran the project.
Will make some improvements in the readme.
